.github/workflows/android-custom-build.yml

```diff
@@ -1,15 +1,15 @@
 name: ID PASS - MOSIP Resident Application Custom build
 
 env:
-  backendServiceDefaultUrl: https://api-internal.qa4.mosip.net/residentmobileapp
```

Comment:
why is it hard coded in the yml? it should be an input for yml. Look at this https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/ 
Work with chandra and his team if you need clarity.